packages/typespec-ts/package.json

```diff
@@ -50,42 +50,43 @@
   "author": "Jose Heredia <joheredi@microsoft.com>",
   "license": "MIT",
   "devDependencies": {
-    "@types/chai": "^4.3.1",
-    "@types/mocha": "^10.0.6",
-    "@types/node": "^18.0.0",
-    "@types/fs-extra": "^9.0.13",
-    "@typescript-eslint/eslint-plugin": "^6.8.0",
-    "@typescript-eslint/parser": "^6.8.0",
-    "chai": "^4.3.6",
-    "mkdirp": "^3.0.1",
-    "eslint": "^8.9.0",
-    "mocha": "^10.4.0",
-    "rimraf": "^5.0.0",
-    "ts-node": "~10.9.1",
-    "typescript": "~5.4.5",
-    "prettier": "^3.1.0",
-    "@azure-tools/cadl-ranch-specs": "^0.33.4",
+    "@azure-rest/core-client": "^2.0.0",
     "@azure-tools/cadl-ranch-expect": "^0.13.4",
+    "@azure-tools/cadl-ranch-specs": "^0.33.4",
     "@azure-tools/cadl-ranch": "^0.12.8",
-    "chalk": "^4.0.0",
-    "@azure-rest/core-client": "^2.0.0",
+    "@azure-tools/typespec-azure-core": ">=0.42.0 <1.0.0",
+    "@azure-tools/typespec-client-generator-core": ">=0.42.2 <1.0.0",
+    "@azure/abort-controller": "^2.0.0",
     "@azure/core-auth": "^1.6.0",
-    "cross-env": "^7.0.3",
-    "@azure/core-paging": "^1.5.0",
     "@azure/core-lro": "3.0.0-beta.1",
+    "@azure/core-paging": "^1.5.0",
     "@azure/core-rest-pipeline": "^1.14.0",
-    "@azure/abort-controller": "^2.0.0",
-    "@azure/logger": "^1.0.4",
     "@azure/core-util": "^1.4.0",
-    "eslint-plugin-require-extensions": "0.1.3",
-    "@typespec/ts-http-runtime": "1.0.0-alpha.20240314.2",
-    "@azure-tools/typespec-azure-core": ">=0.42.0 <1.0.0",
-    "@azure-tools/typespec-client-generator-core": ">=0.42.2 <1.0.0",
+    "@azure/logger": "^1.0.4",
```

Comment:
I have a local linter configured to sort the package.json entries here. These changes are all constrained to the commit labeled "format", and it's only the last few commits that have substantial changes. It might be easier to review this PR one commit at a time.

Reply:
There's a recent commit that adds a new dependency `@azure-tools/typespec-azure-resource-manager` into this package.json, but it's missing now, not sure if it's because you didn't merge latest main?